forte/data/data_store.py

```diff
@@ -253,6 +254,47 @@ def _new_group(self, type_name: str, member_type: str) -> List:
         entry += len(self._type_attributes[type_name]) * [None]
         return entry
 
+    def _is_subclass(
+        self, type_name: str, cls, include_onto_file: bool = False
+    ) -> bool:
+        r"""This function takes a fully qualified ``type_name`` class name,
+        ``cls`` class and returns whether ``type_name``  class is the``cls``
+        subclass or not. This function accpect two types of class: the class defined
+        in forte, or the classes in user provided ontology file.
+
+        Args:
+            type_name: A fully qualified name of an entry class.
+            cls: An entry class.
+            include_onto_file: A boolean value whether ``type_name`` class
```

Comment:
Let's simply rename this variable to avoid confusion.

```suggestion
            no_dynamic_subclass: A boolean value controlling where to look for subclasses. If True, this function will not check the subclass relations via `issubclass` but rely on pre-populated states only.
```